app/build.gradle

```
@@ -6,7 +6,7 @@ apply plugin: 'kotlin-kapt'
 apply from: '../dependencies.gradle'
 
 android {
-    compileSdkVersion versions.targetSdk
+    compileSdkVersion 30
```

```suggestion
    compileSdkVersion versions.targetSdk
```
Seems you can edit this in `dependencies.gradle`